src/main/webapp/course.html

```diff
@@ -134,7 +129,7 @@ <h1 class="p-4 text-center">Add Your Review</h1>
               <p class="lead">Hours/Week on this Course</p>
               <input
                 id="hours-work"
-                name="hoursOfWork"
+                name="hours-work"
```

Comment:
changed this to hours in my version, so for consistency do you want me to change to hours-work or viceversa?

Reply:
I mean id="hours"

Reply:
I can change it to hours